src/UI/citizens/components/header/CitizensHeaderStyle.js

```diff
@@ -1,12 +1,353 @@
 import styled from 'styled-components';
 
 const HeaderContainer = styled.header`
-  width: 100%;
+  width: 1440px;
   padding: 20px;
 `;
 
 const HeaderTitle = styled.h1`
-  color: red;
+  width: 576px;
+  height: 40px;
+  font-family: Lora;
+  font-size: 32px;
+  font-weight: bold;
```

Comment:
Izbaci nepotrebne property-je jer većina njh već ima value "normal" po defaultu. Provjeri sa developer toolsima mjenja li se išta kad nešto izbaciš